### ISSUE TITLE
cannot import existing annotations from GO database into a GO_CAM

### ISSUE THREAD

**Issue by Pauldenny** [geneontology/noctua#928]
![image](https://github.com/user-attachments/assets/ad242ebe-beca-4782-8c73-c47d3bf23f7f)
When trying to import existing annotations from GO database for P25051 e.g. these IEAs
![image](https://github.com/user-attachments/assets/a2026af6-d7e0-4dfa-a266-09622b10fee9)
noctua seems unable to find them.

**Comment by Pauldenny** [comment]
Kimberley @vanaukenk suggested this issue is a result of AMIGO2 including only EXP annotations for some species - would it be possible for these species to be added to the shortlist for which other non-EXP annotations could be included?
I am working on antimicrobial resistance, which can involve many non-reference / non-MOD species:
 
E. coli [taxID=562] - Not just the K12 MOD
Staphylococcus aureus [1280]
Klebsiella pneumoniae (High priority organism - WHO GLASS report 2022) [573]
Acinetobacter baumanii [470]
Pseudomonas aeruginosa - particularly high risk for multi-drug resistance [287]
Enterobacteriaceae [543]

**Comment by pgaudet** [comment]
Hi @Pauldenny 
@vanaukenk correctly identified the issue you are observing - we only have EXP annotations for P25051

https://amigo.geneontology.org/amigo/gene_product/UniProtKB:P25051

I dont think we can load the additional data in the very short term, so you'll have to make that annotation again in Noctua.

WRT strains versus species, I am also in favor of GO central being strain-specific, and moving all 'strain' annotations up to the species. However that involves Uniprot chaning their reference proteomes, as far as I know. 

Again, this is on the radar, but it may take a ltitle while to get done. However these use cases are extremely valuable in deciding which way to go. 

Thanks for your patience - sorry we cannot help more in the immediate future. 

@kltm anything you'd like to add? or correct if I stated anything incorrectly.

**Comment by Pauldenny** [comment]
One point I may not have explained correctly:
When I asked for annotations to these taxa to become accessible in AMIGO2/Noctua, I meant that, ideally, I would want to be able to access / annotate any "descendants" / children of these taxa e.g. taxID=562 has 2,926 direct descendants.

**Comment by pgaudet** [comment]
humm I'll check with @alexsign how this can be done

Do you need to annotate the strains? or are the species ok ? ie, could we map the annotations to the strains up to the species? (note that I dont even know if that's doable ;)

**Comment by Pauldenny** [comment]
@pgaudet yes - I'm not sure this could be done - UniProt is also about to change it's rules on importing new Proteome sets. However, reviewed entries already in Swiss_prot shouldn't be affected by the proteome changes.

Just hit a possibly related glitch:
I can see this in AMIGO https://amigo.geneontology.org/amigo/gene_product/UniProtKB:P52477
But the UniProt accession isn't recognised by Noctua.
Instead Noctua seems to be using a synonymous identifier derived from an Ordered locus name ID - PA0425

![image](https://github.com/user-attachments/assets/c9070f48-6ab0-4279-a0a7-cff4ca259682)
Confusing - and Noctua still doesn't find GO annotations displayed by AMIGO...

**Comment by kltm** [comment]
@Pauldenny The entities found in the autocomplete of Noctua and the annotations found in AmiGO have slightly different derivations, which is what I think is causing `UniProtKB:P52477` to not show up in the autocomplete.
Grossly: these IBAs in AmiGO are coming in from PANTHER; the entities in the Noctua autocomplete are coming in from either self-created files by the resources for entities that they desire to annotate or (if that's not available) the upstream (pre-IBA) GAF from the resource.

**Comment by thomaspd** [comment]
@pgaudet can we close this issue?

**Comment by pgaudet** [comment]
Hi @Pauldenny 
I'm closing this because we have now loaded all the requested bacteria reference organisms , see https://github.com/geneontology/neo/issues/116

We cannot load E coli Taxon:562 because (1) it doesn't have a reference proteome and (2) it would end up loading 481,000 entries, see https://www.uniprot.org/uniprotkb?query=(organism_id:562)

If UniProt makes it a Reference Protome, we'll be able to load it. So I am closing this for now. 

Thanks, Pascale

**Comment by kltm** [comment]
I'd note that, temporarily, taxon:12118 is not part of the load due to an identifier collision. @pgaudet knows about this and a fix is being pursued.
